i18n/Messages_fr.properties

```diff
@@ -18,7 +18,7 @@ msgstr ""
 
 #: Applications/Games/League of Legends/BETA Client/script.js:41
 msgid "Select your region:"
-msgstr ""
+msgstr "Sélectionnez votre région :"
```

Comment:
There seems to be a ` ` (space) character to much after `région`

Reply:
Can you fix it?